get_dsp.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
```

Comment:
can this file be renamed to input_parameters or something else? 

DSP is not very indicative of what its doing. dsp also distracting personally to me as it stands for digital signal processing, and I was expecting a module along that lines

Reply:
I don't think that file is used, given that it's only 7 lines and looks very incomplete, so I deleted it